Workbooks/Workloads/NGINX Insights/NGINX Insights.workbook

```diff
@@ -1080,7 +1080,7 @@
                   "name": "SuggestedAlerts",
                   "type": 1,
                   "isRequired": true,
-                  "value": "[     {         \"AlertName\": \"host-high-cpu\",         \"AlertDescription\": \"Fires when high CPU usage is detected on the host\",         \"AlertThreshold\": 70,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Cpu/azuredeploy.json\"     },     {         \"AlertName\": \"host-used-memory-percentage\",         \"AlertDescription\": \"Fires when Used Memory percentage on the host exceeds a threshold\",         \"AlertThreshold\": 85,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Memory/azuredeploy.json\"     } ]",
+                  "value": "[     {         \"AlertName\": \"host-high-cpu\",         \"AlertDescription\": \"Fires when high CPU usage is detected on the host\",         \"AlertThreshold\": 70,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Cpu/azuredeploy.json\"     },     {         \"AlertName\": \"host-used-memory-percentage\",         \"AlertDescription\": \"Fires when Used Memory percentage on the host exceeds a threshold\",         \"AlertThreshold\": 85,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Memory/azuredeploy.json\"     },     {         \"AlertName\": \"NGINX ratio of active requests to handled\",         \"AlertDescription\": \"Fires if ratio of active requests to handled exceeds threshold\",         \"AlertThreshold\": 1,         \"AlertTemplate\": \"https://azmonsolutions.blob.core.windows.net/workloadinsights/Alerts/NGINX/ratio-active-requests-to-handled.json\"     } ]",

```

Comment:
For the last alert, is ratio range {0..1} or {0.100}? Either way, the threshold seems odd.

Reply:
Yes agree, need to have consistent types of parameters, probably Percent parameters (?)
and also I don't know how to generate this condition, @austonli  did not either.
Do you know how?

Reply:
Changed to percent based, default is 100